src/back/space/src/model/user_serialize.cpp

```diff
@@ -0,0 +1,42 @@
+#include "user_serialize.hpp"
+#include <boost/uuid/uuid_io.hpp>
+#include <userver/formats/json/value_builder.hpp>
+#include <userver/utils/boost_uuid4.hpp>
+
+namespace svetit::space::model {
+
+formats::json::Value Serialize(
+	const SpaceUser& su,
+	formats::serialize::To<formats::json::Value>)
+{
+	formats::json::ValueBuilder builder{};
+
+	builder["spaceId"] = boost::uuids::to_string(su.spaceId);
+	builder["userId"] = su.userId;
+	builder["isOwner"] = su.isOwner;
+	const auto joinedAt = std::chrono::duration_cast<std::chrono::seconds>(su.joinedAt.time_since_epoch()).count();
+	builder["joinedAt"] = joinedAt;
```

Comment:
Кажется нет смысла выносить в отдельную переменную если всё равно не стало ни короче ни чище.
```suggestion
	builder["joinedAt"] = std::chrono::duration_cast<std::chrono::seconds>(su.joinedAt.time_since_epoch()).count();
```

Либо можно куда нибудь в shared закинуть функцию вроде TimeToSeconds
И тогда тут можно было бы сократить, до 
```cpp
builder["joinedAt"] = utils::TimeToSeconds(su.joinedAt);
```

Reply:
Готово (пока сделал без функции в shared, показалось, что новая функция с 1 строчкой - это на грани лишнего; но если считаешь, что лучше будет именно так через shared функцию, то давай сделаем).